plugin/core/input_handlers.py

```diff
@@ -16,7 +17,7 @@
 P = ParamSpec('P')
 
 
-def debounced(user_function: Callable[P, Any]) -> Callable[P, None]:
+def debounced(user_function: 'Callable[P, Any]') -> 'Callable[P, None]':
```

Comment:
Can't use the `Callable[P, returntype]` syntax anymore, because it was introduced in Python 3.10 ([ParamSpec](https://typing.readthedocs.io/en/latest/spec/generics.html#paramspec)) and the `Callable` which was used here was just a "fake" class.

Reply:
This workaround is not required anymore with the `ParamSpec` imported from typing_extensions.